### ISSUE TITLE
Getting `‘visibility’ attribute ignored` warnings

### ISSUE THREAD

**Issue by TimPushkin** [SRombauts/SQLiteCpp#412]
When SQLiteCpp is built with GCC, a lot of warnings like these arise:
```
/home/runner/work/SQLiteCpp/SQLiteCpp/src/Column.cpp:21:25: warning: ‘visibility’ attribute ignored [-Wattributes]
   21 | SQLITECPP_API const int INTEGER   = SQLITE_INTEGER;
      |                         ^~~~~~~
/home/runner/work/SQLiteCpp/SQLiteCpp/src/Column.cpp:22:25: warning: ‘visibility’ attribute ignored [-Wattributes]
   22 | SQLITECPP_API const int FLOAT     = SQLITE_FLOAT;
      |                         ^~~~~
/home/runner/work/SQLiteCpp/SQLiteCpp/src/Column.cpp:23:25: warning: ‘visibility’ attribute ignored [-Wattributes]
   23 | SQLITECPP_API const int TEXT      = SQLITE_TEXT;
      |                         ^~~~
/home/runner/work/SQLiteCpp/SQLiteCpp/src/Column.cpp:24:25: warning: ‘visibility’ attribute ignored [-Wattributes]
   24 | SQLITECPP_API const int BLOB      = SQLITE_BLOB;
      |                         ^~~~
/home/runner/work/SQLiteCpp/SQLiteCpp/src/Column.cpp:25:25: warning: ‘visibility’ attribute ignored [-Wattributes]
   25 | SQLITECPP_API const int Null      = SQLITE_NULL;
      |                         ^~~~
```

The above is copied from SQLiteCpp's own CI on Ubuntu.

**Comment by ninjaoflight** [comment]
should be fixed in #413

**Comment by SRombauts** [comment]
Thanks a lot @ninjaoflight
